.eslintrc

```diff
@@ -32,7 +32,8 @@
     "semi": [1, "always"],
     "no-trailing-spaces": 0,
     "eol-last": 0,
-    "no-unused-vars": 0,
```

Comment:
Please do add this line to your .eslintrc - it will help you protect against problems and bugs.